packages/@aws-cdk/aws-certificatemanager/test/certificate.test.ts

```diff
@@ -91,6 +91,37 @@ test('throws when domain name is longer than 64 characters', () => {
   }).toThrow(/Domain name must be 64 characters or less/);
 });
 
+test('does not throw when domain name is longer than 64 characters with tokens', () => {
```

Comment:
Unless the code performs validation on the `domainName` beyond just the length of it, please simplify the string as I've suggested below.